### PR TITLE
Move replace cards button

### PR DESCRIPTION
**Pull request by Geor23**

### CODE REVIEW

File: client/includes/templates/game/game_question.html [hunk 1]
```diff
@@ -15,6 +15,11 @@ <h2 animate-on-change="dynamicFilledInQuestion()" ng-bind-html="bindHtml(dynamic
                     </button>
                 </div>
             </div>
+            <div class="col-md-6 col-sm-6 col-xs-12 text-center">
+            <!-- Spaceing for extra small screens between buttons when stacked -->
+                <div class="hidden-lg hidden-md hidden-sm">&nbsp;</div>
+                <button id="replace-hand-button" type="button" class="btn btn-lg btn-success" ng-click="replaceHand(userHand())">Replace hand <span class="badge">-{{handReplaceCost()}} points</span></button>
```
Comment (JamesAdamHughes) [inline client/includes/templates/game/game_question.html]: can this be red like dave suggested?

**Comment by BRHoward** [comment]
:+1:

**Comment by BRHoward** [comment]
:large_blue_diamond:

**Comment by JamesAdamHughes** [comment]
noice

**Pull request merged**